server/proxy/install.go

```diff
@@ -93,30 +107,31 @@ func (p *Proxy) InstallApp(in Incoming, cc apps.Context, appID apps.AppID, deplo
 		return nil, "", err
 	}
 
-	var message string
+	message := fmt.Sprintf("Installed %s.", app.DisplayName)
 	if app.OnInstall != nil {
-		resp := p.callApp(in, *app, apps.CallRequest{
-			Call:    *app.OnInstall,
-			Context: cc,
-		})
-		// TODO fail on all errors except 404
-		if resp.Type == apps.CallResponseTypeError {
-			log.WithError(err).Warnf("OnInstall failed, installing app anyway.")
-		} else {
-			message = resp.Markdown
+		cresp := p.call(in, *app, *app.OnInstall, &cc)
+		if cresp.Type == apps.CallResponseTypeError {
+			// TODO: should fail and roll back.
+			log.WithError(cresp).Warnf("Installed %s, despite on_install failure.", app.AppID)
+			message = fmt.Sprintf("Installed %s despite on_install failure: %s", app.AppID, cresp.Error())
```

Comment:
To be consistent with the message above

```suggestion
			message = fmt.Sprintf("Installed %s, despite on_install failure: %s", app.AppID, cresp.Error())
```